app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt

```diff
@@ -221,5 +224,8 @@ class DashboardActivity : MifosBaseActivity(), View.OnClickListener, NavigationV
         val intent = Intent(this, AboutUsActivity::class.java)
         startActivity(intent)
     }
+    fun showFaq(){
+        Toast.makeText(applicationContext,"FAQ section yet to be added",Toast.LENGTH_SHORT).show()
```

Comment:
hey. kindly use the ```Toaster``` class in utils :) check line 215 for reference.

Reply:
Done ;)